src/main/java/Storage.java

```diff
@@ -0,0 +1,84 @@
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static final String FILEPATH = "data/savedData.txt";
+    private static final String SAVEDIR = "data";
+
+    public static void saveData(ArrayList<Task> tasks) throws IOException{
+        StringBuilder line = new StringBuilder();
+        for(Task task: tasks){
```

Comment:
Need to add spaces between `for` and the bracketed condition, and the `{`, likewise for other similar parts of the code with control flow statements